src/test/java/PlayerInfoTest.java

```diff
@@ -0,0 +1,383 @@
+
+
+import static org.junit.Assert.*;
+
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.openRealmOfStars.AI.PathFinding.PathPoint;
+import org.openRealmOfStars.player.PlayerInfo;
+import org.openRealmOfStars.player.SpaceRace.SpaceRace;
+import org.openRealmOfStars.player.fleet.Fleet;
+import org.openRealmOfStars.player.ship.ShipDesign;
+import org.openRealmOfStars.player.ship.ShipStat;
+import org.openRealmOfStars.player.ship.generator.ShipGenerator;
+import org.openRealmOfStars.player.tech.Tech;
+import org.openRealmOfStars.player.tech.TechFactory;
+import org.openRealmOfStars.player.tech.TechList;
+import org.openRealmOfStars.player.tech.TechType;
+import org.openRealmOfStars.starMap.Coordinate;
+import org.openRealmOfStars.starMap.StarMap;
+import org.openRealmOfStars.starMap.Sun;
+import org.mockito.Mockito;
+

```

Comment:
Here should be the copyright text and GPL license. Seems that this projects needs an AUTHORS.md file and team name which would hold the copyrights.